.gitattributes

```diff
@@ -0,0 +1,24 @@
+# Auto detect text files and perform LF normalization
+*        text=auto
+
+*.java   text
```

Comment:
Currently, `.java` files use cl-rf in this repo. Keeping this as the standard would keep more of the `git blame history` intact.

Reply:
Updated. Though there were still some files for which the line endings changed.